.github/workflows/main.yml

```diff
@@ -70,6 +70,35 @@ jobs:
           file: build/*/*.dmg
           tag: ${{ github.ref }}
           file_glob: true
+          
+  appcast:
+    name: Set Appcast
+    needs: build
+    runs-on: macos-latest
+    env:
+      JOB_TYPE: BUILD
+      SIGNATURE_FOR_SIGNING: ${{ secrets.SIGNATURE_FOR_SIGNING }}
+    if: github.event_name == 'release'
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Get Tag
+        run: echo "TAG_VER=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
```

Comment:
Is there any particular reason to expose `TAG_VER` through GitHub environment? I only see it being used in shell scripts below. Sounds like it can be just a normal variable.

Reply:
Fixed.